python/pyspark/__init__.py

```diff
@@ -51,6 +51,26 @@
 # for back compatibility
 from pyspark.sql import SQLContext, HiveContext, SchemaRDD, Row
 
+
+def since(version):
```

Comment:
OK - no problem - I'll update this (+ SPARK-10269/10271/10272) once SPARK-10373 is complete.